index.html

```diff
@@ -8,7 +8,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
 		<script src="./review-buttons/init.js"></script>
-		<link rel="stylesheet" href="style.css" />
+		<link rel="stylesheet" href="/style.css" />
```

Comment:
I believe you should start with ./ in this case. Otherwise style might not work in some of the browsers 
<link rel="stylesheet" href="./style.css" />